reproman/support/jobs/orchestrators.py

```diff
@@ -931,11 +942,12 @@ def fetch(self, on_remote_finish=None):
         ref = self.job_refname
         lgr.info("Updating local dataset with changes from '%s'",
                  resource_name)
-        self.ds.repo.fetch(resource_name, "{0}:{0}".format(ref),
-                           recurse_submodules="no")
+        repo = self.ds.repo
+        repo.fetch(resource_name, "{0}:{0}".format(ref),
+                   recurse_submodules="no")
         failure = False
         try:
-            self.ds.repo.merge(ref)
+            repo.merge(ref)
         except DCError as exc:
             lgr.warning(
```

Comment:
so it is just a warning... shouldn't `failure = True` be set?

Reply:
Yeah, it'd make sense to set `failure = True`, because it'd prevent trying the `.get` call on outputs, which would presumably fail or at least not be useful if we failed to bring in the target tree.  However, I don't think it matters much one way or the other once we're in this unexpected error state, and I would prefer it not be done as part of this very unrelated pr.